terraform/modules/shared/main.tf

```diff
@@ -27,7 +27,7 @@ resource "azurerm_servicebus_namespace" "ingestion" {
   sku                 = "Standard"
 }
 
-resource "azurerm_servicebus_topic" "ingestion" {
+resource "azurerm_servicebus_queue" "ingestion" {
```

Comment:
I see.  Azure uses a servicebus, let's make sure to use that lingo in the comments/documentation above.  I know I will need to update this in the quick-start-guide.